plugins/pdr_deterministic_plugin/tests/simulation_telemetry.py

```diff
@@ -130,31 +193,69 @@ def start_server(port:str,changes_intervals:int, run_forever:bool):
                 last_val = counter['last_val']
                 ## here we set the value for the counters
                 if ENDPOINT_CONFIG["ITERATION_TIME"] < MAX_ITERATIONS:
-                    counter['last_val'] = findValue(index,counters_names[i],ENDPOINT_CONFIG["ITERATION_TIME"])
+                    counter['last_val'] = find_value(index,counters_names[i],ENDPOINT_CONFIG["ITERATION_TIME"], 0)
                 else:
-                    counter['last_val'] = randomizeValues(counters_names[i],ENDPOINT_CONFIG["ITERATION_TIME"])
+                    counter['last_val'] = randomize_values(counters_names[i],ENDPOINT_CONFIG["ITERATION_TIME"])
                 row_data.append(str(last_val))
             data.append(row_data)
 
         output = [header] + data
         csv_data = '\n'.join([','.join(row) for row in output]) + '\n'
-        endpoint['data'] = csv_data        
+        endpoint['data'] = csv_data
         if not run_forever and ENDPOINT_CONFIG["ITERATION_TIME"] > MAX_ITERATIONS:
             # after all the tests are done, we need to stop the simulator and check the logs
             return
         time.sleep(changes_intervals)
 
+def black_ports_simulation(endpoint):
```

Comment:
Please rename the method to excluded_ports_simulation

Reply:
Done. Some comments are changed too.